### PR TITLE
specify build pkg dir when build rpm/deb

### PR DESCRIPTION
**Pull request by xuweibj**
Add env "BUILDDESTDIR", if not null, will copy core tarball to this folder.

### CODE REVIEW

File: build-ubunturepo [hunk 1]
```diff
@@ -400,6 +401,16 @@ __EOF__
     chgrp root $tar_name
     chmod g+w $tar_name
 
+    if [ "$PKGDIR" ]; then
```
Comment (neo954) [inline build-ubunturepo]: I think it may need to validation user input before use it. In this way, `$PKGDIR` can be anything. It is not safe.

Comment (neo954) [reply build-ubunturepo]: Say, pass `PKGDIR=/dev/mem` will ruin the system.

File: build-ubunturepo [hunk 2]
```diff
@@ -26,6 +26,7 @@
 #                            
 #               LOG=<filename> - provide an LOG file option to redirect some output into log file
 #
+#               PKGDIR=<build_pkg_dir> - Copy build core tarball to PKGDIR if defined
```
Comment (immarvin) [inline build-ubunturepo]: this variable should better be `DESTDIR`

File: build-ubunturepo (hunk 1 above)
Comment (immarvin) [reply build-ubunturepo]: do we need to remove any existing `$PKGDIR` ?

**Comment by immarvin** [comment]
do you need to put xcat dependencies to the specified path?

**Comment by immarvin** [comment]
i am ok with the change

**Pull request merged**